tests/Behat/Provider/Calendar.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Calendar\Tests\Behat\Provider;
+
+use Sylius\Calendar\Provider\DateTimeProviderInterface;
+
+final class Calendar implements DateTimeProviderInterface
+{
+    private string $projectDirectory;
+
+    public function __construct(string $projectDirectory)
+    {
+        $this->projectDirectory = $projectDirectory;
+    }
+
+    public function now(): \DateTimeInterface
+    {
+        return $this->provideFakeDateIfSet();
+    }
+
+    private function provideFakeDateIfSet(): \DateTimeInterface
+    {
+        if (file_exists($this->projectDirectory . '/var/temporaryDate.txt')) {
+            $file = fopen($this->projectDirectory . '/var/temporaryDate.txt', 'r');
+            $dateTime = fgets($file);
+            fclose($file);
+
+            return new \DateTimeImmutable($dateTime);
+        }
+
+        return new \DateTimeImmutable();
```

Comment:
I believe if the file does not exist it should throw an exception - reading the date from the file is the only reason of this service, this way i pretends it works when it does not 🔥